docs/core/compatibility/aspnet-core/6.0/razor-pages-logging-ids.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Breaking change: Razor: Logging ID changes"
+description: "Learn about the breaking change in ASP.NET Core 6.0 where some Razor pages logging IDs were changed."
+no-loc: [ Razor ]
+ms.date: 09/01/2021
+---
+# Razor: Logging ID changes
```

Comment:
@pranavkm The announcement mentioned name changes, but in the old/new behavior section, I didn't see any name changes. So I left that out of the title and description.

Reply:
Ahh, I copied the message verbatim from a different issue, but forgot to remove this bit. We only updated the event Ids as part of this change. Thanks for bringing this up!